docs/troubleshooting.md

```diff
@@ -98,6 +98,142 @@ for p in $(oc get pods -l component=es -o jsonpath={.items[*].metadata.name}); d
   oc exec -c elasticsearch $ES_POD -- touch /opt/app-root/src/init_failures;  \
 done
 ```
+### Elasticsearch performance degrades around 00:00 UTC
+This situation is a result of Elasticsearch processing bulk index requests from all the nodes in the OKD cluster for
+logs for the new day.  Creation of a significant number of new indices simultaneously can consume significant resources when there are a large number of projects.  Periodically pre-creating the next day's indices can reduce the load on ElasticSearch when rolling over to the new day:
```

Comment:
s/logs for the new day./logs for the new day where Elasticsearch auto-creates new indices for those logs./

Reply:
Consider re-writing:

    Creation of a significant number of new indices simultaneously
    can consume significant resources when there are a large
    number of projects.

To something like:

    Creation of a significant number of new indices simultaneously
    can cause bulk index processing to slow down as it waits for all
    members of the cluster to become aware of each new index.  This
    is likely to happen when there are many active projects (namespaces)
    in a cluster, since each projects has its own index.